scripts/hurricane/stats/exevs_hurricane_regional_tropcyc_stats.sh

```diff
@@ -109,7 +109,7 @@ export enddate="$YY02$MM02$DD02$HH02"
 echo "$startdate, $enddate"
 
 #--- run for TC_pairs
-cp ${PARMevs}/metplus_config/hurricane/stats/TCPairs_template.conf .
+cp ${PARMevs}/TCPairs_template.conf .
 export SEARCH0="METBASE_template"
```

Comment:
You will want to get this change back into your PR. You accidentally introduced older code for line 112.